### DISCUSSION TITLE
It's possible to use custom roms?

### DISCUSSION THREAD

**Discussion by artur6768** [86Box/86Box#2684] [Q&A] [answered]
Hi 86box community, recently I found a Brazilian IBM XT clone, and I want to test it, however I wasn't able to add it as a machine using 86box manager.
  So my question is: it's possible to use a custom rom in 86box ? if so, how can I add it?

Thanks, Artur6768

============Update==========
After searching a bit more, I found that if you rename the original and overwrite to the clone, it might work, and after doing this, I managed to boot the IBM XT clone.
So my new question is: is there a way to add this ROM without overwriting the original one?
![imagen](https://user-images.githubusercontent.com/96083801/190924290-80ec7f6e-5b59-44a9-8a54-e78f33d8ef42.png)

Thanks, Artur6768

**Comment by dhrdlicka** [comment, marked as answer]
No, there is no way to add new ROMs without modifying the source code and recompiling.
